five-phases/README.md

```diff
@@ -0,0 +1,26 @@
+Design Thinking is User-Centric.  Therefore the most important characteristic of a solution is that it meets the needs of the User.  To ensure solutions achieve this objective, the three fundamental principles of Design Thinking guide teams through the process of defining the MVP, and then iteratively improving and enhancing it to achieve the final product.
+
+These three principles are:
+1. Empathy
+2. Ideation
+3. Experimentation
+
+**Empathy** is simply understanding the needs of the user.  A designer must set aside their own assumptions about the problem so that they might gain this insight.  In Design Thinking, the team must always remain focused on the people for whom they are designing.
```

Comment:
**Empathy** is simply understanding the needs of the user.  A designer must set aside their own assumptions about the problem so that they might gain this insight.  In Design Thinking, the team must always remain focused on the people for whom they are designing.